maps/_map.dm

```diff
@@ -101,3 +107,34 @@ var/global/list/accessable_z_levels = list()
 	return
 */
 
+// Base Turf //////////////////////////////////////////////////
+
+//Returns the lowest turf available on a given Z-level, defaults to space.
+var/global/list/base_turf_by_z = list()
+
+proc/get_base_turf(var/z)
+	if(!base_turf_by_z["[z]"])
+		base_turf_by_z["[z]"] = /turf/space

```

Comment:
get_base_turf searches for what would otherwise be the under_turf though

What I don't know is how I look through the Z-level datums with the Z coordinates